src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Fruit> fruits = new ArrayList<>();
+    public static final List<String> stringList = new ArrayList<>();
+    public static final Fruit banana = new Fruit("banana", 0);
+    public static final Fruit apple = new Fruit("apple", 0);
```

Comment:
```suggestion
    public static final Map<String, Integer> storage = new HashMap<>();
```
We should be able to keep any kind of fruit - not only bananas and apple 
Storage for fruit only